Firestore/Source/Remote/FSTSerializerBeta.mm

```diff
@@ -937,43 +939,44 @@ - (GCFSStructuredQuery_Filter *_Nullable)encodedFilters:(NSArray<FSTFilter *> *)
   return result;
 }
 
-- (GCFSStructuredQuery_Filter *)encodedRelationFilter:(FSTRelationFilter *)filter {
+- (GCFSStructuredQuery_Filter *)encodedRelationFilter:
+    (const std::shared_ptr<RelationFilter> &)filter {
```

Comment:
Can you simply take a const reference to a `RelationFilter` here? This function doesn't seem to be concerned with ownership.

Reply:
Personally, I like the guideline that functions not concerned with ownership should not have smart pointers in their signature. To me, conveying this information is more valuable than symmetry.